features/consent/use-allow-group.mutation.ts

```diff
@@ -121,7 +119,7 @@ export const getAllowGroupMutationOptions = (
         group.topic,
         context.previousConsent || "unknown"
       );
-      updateConversationInConversationsQuery({
+      updateConversationInConversationsQueryData({
```

Comment:
metadata or data?

Reply:
data. It's just how i always name those function. To say that I'm updating the data of the query

Reply:
perfect just clarifying

Reply:
perfect just clarifying